pkg/release/extract/extract-tf.go

```diff
@@ -0,0 +1,665 @@
+package extract
```

Comment:
This code is almost all cut-n-paste from https://github.com/openshift/oc/blob/master/pkg/cli/admin/release/extract_tools.go

commit Marked as WIP because:
- we can probably simplify more 
- rename
- debug log messags need to be removed